docs/core_docs/docs/integrations/llms/arcjet.mdx

```diff
@@ -0,0 +1,34 @@
+import CodeBlock from "@theme/CodeBlock";
```

Comment:
We prefer standardization along the lines of this template:

https://github.com/langchain-ai/langchainjs/blob/main/libs/langchain-scripts/src/cli/docs/templates/llms.ipynb

Happy to update it though after other comments are addressed

Reply:
I've migrated this to jupyter